pom.xml

```diff
@@ -230,6 +230,7 @@
         <xml-matchers.version>0.10</xml-matchers.version>
         <zip4j.version>1.3.2</zip4j.version>
         <zookeeper.version>3.4.13</zookeeper.version>
+        <owasp-java-encoder>1.2.2</owasp-java-encoder>
```

Comment:
Should be in alphabetical order, this is out of place

Reply:
Property should also end in `.version` so something like `owasp-java-encoder.version`

Reply:
I did not realize we were doing props in alphabetical order, previously we used to add new properties at the bottom, I'll change it to make it in order.

Missed the .version, forgot to clean up, I'll fix this.

Reply:
Yea alphabetical after I cleaned up all the poms. Makes it easier to find later :)